### DISCUSSION TITLE
Cell types and Cell states? What's difference?

### DISCUSSION THREAD

**Discussion by PWbio** [mathewchamberlain/Signac#7] [Q&A] [answered]
May I ask what is the difference between **cell type** and **cell states**?

I noticed that most of them are similar but a few of them show difference.

for example, a cell can be labeled as:

CellTypes: `Plasma cells`
CellStates: `Epithelial`

Which one should I trust?

Moreover, I see some cells are labeled as:
Immune: `Immune`
CellTypes: `NonImmune`
CellStates: `NonImmune`

I wonder how these labels are generated. And, which label column should I use?

---- update ----
Found a [paper](https://genome.cshlp.org/content/25/10/1491.full.html) about cell type and cell state. Relevant [website](https://mbernste.github.io/posts/cell_types_cell_states/).

**Comment by savova** [comment, marked as answer]
We use the tern cell states to refer to functional subtypes of broad cell types. Cell types generally differ from each other by macro characteristics (e.g. size, shape) and diverge early in lineage. In contrast, cell states refer more to a functional specialization within a cell type.  E.g. T-reg is regulatory functional state of T cells.
